content/vault/v1.21.x (rc)/content/api-docs/system/internal-counters.mdx

```diff
@@ -1292,6 +1292,13 @@ it may be up to 20 minutes delayed.
 - This endpoint can be called from all namespaces. The requested namespace will act as a filter. The exported data will include activity for the requested
   namespace and all of its children.
 
+- `token_creation_time` in the response is the creation time of the token that was used by the client to perform the request. This time is distinct from the time the client was created and may lie outside of the queried time period.
+   <Note>
+   This field was renamed to token_creation_time in Vault 1.21. In previous versions, it was called timestamp.
+  </Note>
+
+- `client_first_usage_time` in the response is a timestamp that represents the earliest usage of the client during the queried time period.
+
 - **`sudo` required** – This endpoint requires `sudo` capability in addition to
```

Comment:
Suggest deleting the sudo line in favor of using the restricted endpoint partial directly below the "Activity export" heading